### PR TITLE
Add small guides for the community and fix some typos.

### PR DESCRIPTION
**Pull request by herulume**
After accepting you should update the master as well.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or advances
```
Comment (JoseFilipeFerreira) [inline CODE_OF_CONDUCT.md]: He says while having commits with those names

Comment (herulume) [reply CODE_OF_CONDUCT.md]: The commits will be squashed and that code of conduct will only be in effect AFTER this is accepted.

File: lib/Listerine/commands.ex [hunk 2]
```diff
@@ -3,9 +3,9 @@ defmodule Listerine.Commands do
 
   @prefix "$"
   @command_desc [
-    study: "Adiciona-te aos channels das cadeiras.",
-    unstudy: "Remove-te dos channels das cadeiras.",
-    mancourses: "Apresenta informação sobre como aceder aos channels das cadeiras.",
+    study: "Adiciona-te as salas das cadeiras.",
```
Comment (JoseFilipeFerreira) [inline lib/Listerine/commands.ex]: a tradução deveria ser `às salas` e não `as salas`

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (JoseFilipeFerreira) [reply CODE_OF_CONDUCT.md]: Makes sense to me

File: lib/Listerine/helpers.ex [hunk 3]
```diff
@@ -5,9 +5,10 @@ defmodule Listerine.Helpers do
     do: "https://cdn.discordapp.com/icons/" <> guild.id <> "/" <> guild.icon <> ".png"
 
   @doc """
-  Returns the intersection of the two sets
+  Returns the intersection of the two sets.
+  Note: It's right associative.
   """
-  def intersect(a, b), do: a -- (a -- b)
+  def intersect(a, b), do: a -- a -- b
```
Comment (Mendess2526) [inline lib/Listerine/helpers.ex]: :fearful:

Comment (JoseFilipeFerreira) [reply lib/Listerine/helpers.ex]: 😨

**Pull request merged**